src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.TaskList;
+
+public class FindCommand extends Command {
+    public static final String COMMAND_WORD = "find";
+    public static final String MESSAGE_USAGE = "find <keyword>";
+    public static final String MESSAGE_EMPTY_KEYWORD = "you are missing a keyword to find by!";
+    private final String keyword;
+
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public String[] execute() {
```

Comment:
It would be nice if this `execute` method had a JavaDoc comment like the corresponding methods in other subclasses of `Command`, especially since it's relatively complex